tests/test_commands.py

```diff
@@ -126,3 +135,41 @@ def test_output_listing(mapdl, plastic_solve, func, args):
     if HAS_PANDAS:
         out_df = out.to_dataframe()
         assert isinstance(out_df, pd.DataFrame) and not out_df.empty
+
+@pytest.mark.parametrize('func', ['dlist', 'flist'])
+def test_bclist(mapdl, beam_solve, func):
+    mapdl.mute = True
+    mapdl.finish()
+    mapdl.clear()
+    mapdl.input(examples.verif_files.vmfiles["vm10"])
+
+    mapdl.post1()
+    mapdl.set(1, 2)
```

Comment:
Merely using the `beam_solve` fixture should be sufficient to setup this method. Otherwise, we have to perform a solve each time, making for a long unit test.

Recommending removal of solution code and keep it in the `beam_solve` fixture.

Reply:
Good idea

Reply:
I dont know how that escaped my attention. That block should not be there.